bundles/core/org.openhab.core.jsr223/src/main/java/org/openhab/core/jsr223/internal/engine/scriptmanager/Script.java

```diff
@@ -68,52 +78,133 @@ public Script(ScriptManager scriptManager, File file) throws FileNotFoundExcepti
+						+"TriggerType 			= Java.type('org.openhab.core.jsr223.internal.shared.TriggerType'),\n"
+						+"PersistenceExtensions	= Java.type('org.openhab.core.persistence.extensions.PersistenceExtensions'),\n"
+						+"pe					= Java.type('org.openhab.core.persistence.extensions.PersistenceExtensions'),\n"
+						+"oh 					= Java.type('org.openhab.core.jsr223.internal.shared.Openhab'),\n"
+						+"State 				= Java.type('org.openhab.core.types.State'),\n"
+						+"Command 				= Java.type('org.openhab.core.types.Command'),\n"
+						+"DateTime 				= Java.type('org.joda.time.DateTime'),\n"
+						+"StringUtils 			= Java.type('org.apache.commons.lang.StringUtils'),\n"
+						+"URLEncoder 			= Java.type('java.net.URLEncoder'),\n"
+
+						+"CallType 				= Java.type('org.openhab.library.tel.types.CallType'),\n"
+						+"DateTimeType 			= Java.type('org.openhab.core.library.types.DateTimeType'),\n"
+						+"DecimalType 			= Java.type('org.openhab.core.library.types.DecimalType'),\n"
+						+"HSBType 				= Java.type('org.openhab.core.library.types.HSBType'),\n"
+						+"IncreaseDecreaseType 	= Java.type('org.openhab.core.library.types.IncreaseDecreaseType'),\n"
+						+"OnOffType 			= Java.type('org.openhab.core.library.types.OnOffType'),\n"
+						+"OpenClosedType 		= Java.type('org.openhab.core.library.types.OpenClosedType'),\n"
+						+"PercentType 			= Java.type('org.openhab.core.library.types.PercentType'),\n"
+						+"PointType 			= Java.type('org.openhab.core.library.types.PointType'),\n"
+						+"StopMoveType 			= Java.type('org.openhab.core.library.types.StopMoveType'),\n"
+						+"UpDownType 			= Java.type('org.openhab.core.library.types.UpDownType'),\n"
+						+"StringType 			= Java.type('org.openhab.core.library.types.StringType'),\n"
+						
+						//No. As of now, Nashorn does not support calling super class methods.
+						//http://nashorn-dev.openjdk.java.narkive.com/VX59ksgk/calling-super-methods-when-extending-classes
+						//therefore:
+						+"BusEvent 				= Java.type('org.openhab.model.script.actions.BusEvent'),\n"
+						+"be 					= Java.type('org.openhab.model.script.actions.BusEvent'),\n"
+						
+						+"transform 			= oh.getAction('Transformation').static.transform,\n"
+						
+						//Item
+						+"getItem 				= ItemRegistry.getItem,\n"
+						+"postUpdate 			= BusEvent.postUpdate,\n"
+						+"sendCommand 			= BusEvent.sendCommand,\n"
+						
+						//System
+						+"FileUtils 			= Java.type('org.apache.commons.io.FileUtils'),\n"
+						+"File 					= Java.type('java.io.File'),\n"
+						
+						+"ohEngine				= 'javascript';\n"
+						
+						//Helper Functions and Libs eventually later a lib Folder for default (Auto) loaded Libraries
+						//Bas64: https://gist.github.com/ncerminara/11257943
+						//+"load('configurations/scripts/jslib/b64.js');\n"
+						);
+			} catch (ScriptException e) {
+				logger.error("ScriptException in initializeSciptGlobals while importing default-classes: ", e);
+			}
```

Comment:
please extract into it's own initMethod for the sake of readability of this general init method